test/misc.test.js

```diff
@@ -21,7 +21,7 @@ describe('download report errors', () => {
     }, 150000);
 
     test('download csv report from custom tenant', async () => {
-        let result = await cli(['-u', `${url}/app/discover#/view/571aaf70-4c88-11e8-b3d7-01146121b73d`, '-a', 'basic', '-c', credentials,
+        let result = await cli(['-u', `${url}/app/data-explorer/discover#/view/571aaf70-4c88-11e8-b3d7-01146121b73d`, '-a', 'basic', '-c', credentials,
```

Comment:
@joshuali925 not familiar with this tool - this isn't breaking any behavior right? Just saying if I generate report on this URL I am expecting something

Reply:
it's test so shouldn't break, but this might need to change to include old and new discover url
https://github.com/opensearch-project/reporting-cli/blob/861b7ea6b6121ee3f019b8737df056ac95469d98/src/download-helpers.js#L190